compiler/rustc_hir_analysis/src/astconv/bounds.rs

```diff
@@ -82,8 +90,8 @@ impl<'tcx> dyn AstConv<'tcx> + '_ {
             );
         }
 
-        if seen_sized_unbound || seen_negative_sized_bound {
-            // There was in fact a `?Sized` or `!Sized` bound;
+        if seen_sized_unbound || seen_negative_sized_bound || seen_positive_sized_bound {
+            // There was in fact a `?Sized`, `!Sized` or `Sized` bound;
             // we don't need to do anything.
         } else if sized_def_id.is_some() {
             // There was no `?Sized` or `!Sized` bound;
```

Comment:
I guess this could be updated, too. I know those comments are super repetitive and obvious. Proceed however you prefer.

Reply:
Done